build.gradle

```diff
@@ -1,5 +1,10 @@
 plugins {
     id 'java'
+    id "application"
+}
+
+application {
+    mainClass = 'org.example.App'
```

Comment:
wondering if instead of `org.example.App` path in folders and classes can be something like `echo.App` or `echoPoC.App`